projects/objects/factory/tools/protos/ScrewHole.proto

```diff
@@ -7,9 +7,10 @@
 PROTO ScrewHole [
   field SFVec3f    translation     0 0 0.2
   field SFRotation rotation        0 0 1 0
-  field SFFloat    radius          0.06
-  field SFFloat    depth           0.1
-  field SFNode     appearance      OldSteel {}
+  field SFFloat    radius          0.06        # Defines the radius of the screw hole.
+  field SFFloat    depth           0.1         # Defines the depth of the screw hole.
+  field SFNode     appearance      OldSteel {} # Defines the appearance of the screw hole.
+  field SFBool     enablePhysics   TRUE        # Defines whether the screw hole should have physics.
```

Comment:
This doesn't seem to be used in this proto...

Reply:
Oops, indeed, it's not even a Solid.